jsonnet/main.jsonnet

```diff
@@ -53,6 +54,7 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                  promLabelProxy: 'quay.io/coreos/prom-label-proxy',
                  kubeRbacProxy: 'quay.io/coreos/kube-rbac-proxy',
                  prometheusAdapter: 'quay.io/coreos/k8s-prometheus-adapter-amd64',
+                 thanos: 'improbable/thanos',
```

Comment:
isn't it quay.io now?

Reply:
Indeed. It was moved to a new org and to quay with the move to the CNCF

Reply:
yes, thanks for catching, I had this on my radar :+1: